pkg/core/mpt/branch.go

```diff
@@ -0,0 +1,76 @@
+package mpt
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/io"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+const (
+	// ChildrenCount represents a number of children of a branch node.
+	ChildrenCount = 17
+	// LastChild is the index of the last child.
+	LastChild = ChildrenCount - 1
+)
+
+// BranchNode represents MPT's branch node.
+type BranchNode struct {
+	hash *util.Uint256
```

Comment:
Do we need to use a pointer here?

Reply:
And in other structures as well. I'd like to minimize microallocations where possible, these structures will be quite heavily used.

Reply:
Fixed.